src/actions/search.js

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {batchActions} from 'redux-batched-actions';
+
+import {Client4} from 'client';
+import {SearchTypes} from 'action_types';
+
+import {forceLogoutIfNecessary} from './helpers';
+import {logError} from './errors';
+import {getProfilesAndStatusesForPosts} from './posts';
+
+export function searchPosts(teamId, terms, isOrSearch = 'false') {
```

Comment:
Are you sure? It looks like it pulls that out as a boolean [here](https://github.com/mattermost/platform/blob/master/api4/post.go#L289)